packages/web-platform/web-elements/src/elements/ScrollView/scroll-view.css

```
@@ -86,13 +86,43 @@ scroll-view[scroll-orientation="horizontal"][enable-scroll="false"] {
     https://chromestatus.com/feature/6752840701706240 It notes that this feature is actively implementing in webkit and gecko.
     chrome 115, firefox no, safari no
   */
+  @keyframes scrollViewTopFading {
```

Stylelint (17.14.0)

[error] Expected keyframe name "scrollViewTopFading" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)